docker/toolkits.txt

```diff
@@ -40,3 +40,4 @@ arcade-outlook-calendar
 arcade-outlook-mail
 arcade-walmart
 arcade-youtube
+arcade-slack_api
```

Comment:
not sure if it will complain or not about the underscore, but `arcade-slack-api` matches what others are listed as